### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by brighton0102**
### In this milestone we :

- Create a collection that keeps a list of books 💯 
- Create a function to add a new book to the collection, with title and author 🚀 
- Create a function to remove a book from the collection

### CODE REVIEW

**Comment by erickma1** [comment]
Fixed all linter errors

File: script.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+// Removing data from local storage
+/* eslint-disable */
+function removeBook(i) {
+  storeData.splice(i, 1);
+  updateData();
+  displayBooks();
+}
+/* eslint-enable */
+window.onload = displayBooks();
```
Comment (codepantha) [inline script.js]: - [ ] Please do not disable linters as they are meant to help you write readable, maintainable and more consistent code while adhering to best practices 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 🎯 
✔️ Descriptive PR
✔️ Feature to add / remove books from a list 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (erickma1) [reply script.js]: @codepantha , Hi, many thanks for pointing out this issue to be resolved.

will it be appropriate to use 

/* eslint-disable no-unused-vars */

Which does not disable linters entirely but indicates to ESLint that the removeBook function is being used in the dynamically generated HTML. This should prevent the ESLint error from appearing.

Comment (erickma1) [reply script.js]: @codepantha

**Comment by erickma1** [comment]
image

changed from  /* eslint-disable */  to /* eslint-disable no-unused-vars */ 

This is to disable linter for  no-unused-vars as described in the Tip of the project requirement.

This does not disable linters entirely but indicates to ESLint that the removeBook function is being used in the dynamically generated HTML. This should prevent the ESLint error from appearing.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @brighton0102 and @codepantha 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**